js/errors.js

```diff
@@ -273,7 +295,7 @@
      * @memberof ERMrest
      * @param {string} message error message
      * @constructor
-     * @desc A no internert was passed to the API.
+     * @desc A no internet was passed to the API.
```

Comment:
This comment should be changed. This error can be for both no internet connection or no connection to the server. Sometimes there are intermittent errors when trying to communicate to the server.